src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/MainConfigurationTest.php

```diff
@@ -74,8 +74,8 @@ public function testCsrfAliases()
             'firewalls' => array(
                 'stub' => array(
                     'logout' => array(
-                        'csrf_provider' => 'a_token_generator',
```

Comment:
I think we should keep a configuration with `csrf_provider` in a legacy test to ensure not to introduce BC breaks in `2.8` patch releases in the future.